src/net/sourceforge/kolmafia/request/PlaceRequest.java

```diff
@@ -340,6 +351,59 @@ private static void parseSotVisit(String responseText) {
     }
   }
 
+  // visible for testing
```

Comment:
Make it `protected` for that reason if you want

Reply:
You can just make it public too! It's the comment I don't like ;)

Reply:
Hangover from a shop where anything that wasn't private had to be documented.  There was an annotation something like @VisibleForTesting but I can't make it work in this environment.  If it is a quirk that only matters to me then I'll delete.